packages/core/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # API Elements: Core
 
+## 0.2.1 (2020-08-27)
```

Comment:
I think this is a breaking change to the contract between the core and the adapters and should be `0.3.0`

Reply:
The `serialize` method works well without passing the attribute `sync`. I'm adding it just to have it explicit.

Reply:
I'm referring more to this case described in https://github.com/apiaryio/api-elements.js/pull/549#discussion_r478458072, adapters designed for @apielements/core 0.2 won't respect `sync: true` (and would return promise). I think from these changes the expectation would be that if sync is set to true, a promise shouldn't be returned.